beta-src/src/utils/map/getOrdersMeta.ts

```diff
@@ -71,13 +80,27 @@ export default function getOrdersMeta(data, phase): Props {
           allowedBorderCrossings = moveChoices.filter((choice) => {
             const { Borders } = choice;
             const from = Borders.find(
-              (border) => border.id === orderUnit.terrID,
+              (border) => border.id === orderUnit?.terrID,
             );
-            if (from && orderUnit.canCrossBorder(from)) {
+            if (from && orderUnit?.canCrossBorder(from)) {
               return true;
             }
             return false;
           });
+          if (phase === "Retreats") {
+            const occupiedTerritory = newBoard.territories.find(
+              (t) => t.id === orderUnit?.terrID,
+            );
+            allowedBorderCrossings = allowedBorderCrossings.filter(
+              (crossing) =>
+                !crossing.unitID &&
+                !crossing.standoff &&
+                crossing.id !== occupiedTerritory?.occupiedFromTerrID,
+            );
+            if (!allowedBorderCrossings.length) {
+              updateOrdersMeta[o.orderData.id].update.type = "Disband";
```

Comment:
Don't update ordersMeta here as it will already be updated in the next lines again.

Reply:
This is for involuntary; if there is no where to go and its in retreats phase, it will immediately become Disbanded, rather than having the option of being disbanded or retreat

Reply:
Oh, this is an object not a function, makes sense.